rules/do-you-know-how-modern-stateless-auth-works/rule.md

```diff
@@ -0,0 +1,67 @@
+---
+type: rule
+title: Do you know how Modern Stateless Authentication works?
+uri: modern-stateless-authentication
+authors:
+  - title: Dhruv Mathur
+    url: https://www.ssw.com.au/people/dhruv-mathur/
+  - title: Gordon Beeming
+    url: https://www.ssw.com.au/people/gordon-beeming/
+  - title: Matt Goldman
+    url: https://www.ssw.com.au/people/matt-goldman/
+created: 2024-03-06T13:42:55.753Z
+guid: fa645fa6-4054-448b-9b4b-d60a66422f1f
+---
+
+Modern stateless authentication, leveraging JSON web tokens (JWT), offers a scalable solution for managing user sessions without server-side storage. The OIDC framework, building on OAuth 2.0, standardizes interactions between authentication services and users. JWTs, containing a header, payload, and signature, ensure data integrity and authentication validity. Issued by trusted identity providers, these tokens facilitate user identification in applications, moving away from traditional session-based methods. This approach enhances scalability and user management, aligning with current web development trends.
+
+<https://www.youtube.com/embed/UBUNrFtufWo>
```

Comment:
Is this how we embed videos? I thought there was a youtube: thing in there :)

Reply:
you are absolutely correct 😄 - I have added that now.